homie-controller/src/lib.rs

```diff
@@ -115,6 +115,7 @@ pub struct HomieController {
     /// The set of Homie devices which have been discovered so far, keyed by their IDs.
     // TODO: Consider using Mutex<im::HashMap<...>> instead.
     devices: Mutex<Arc<HashMap<String, Device>>>,
+    early_property_values: Mutex<Option<HashMap<String, String>>>,
```

Comment:
It would be useful to have a Rustdoc comment here explaining how this field is used.